Formula/bash-snippets.rb

```diff
@@ -1,20 +1,51 @@
 class BashSnippets < Formula
   desc "Collection of small bash scripts for heavy terminal users"
   homepage "https://github.com/alexanderepstein/Bash-Snippets"
-  url "https://github.com/alexanderepstein/Bash-Snippets/archive/v1.16.0.tar.gz"
-  sha256 "19a029a204f16bd4db30cf7fa32aa6953dd10dcb996138b9bc65c01612afaac7"
+  url "https://github.com/alexanderepstein/Bash-Snippets/archive/v1.16.2.tar.gz"
+  sha256 "bbe899bf1b3095761fb405e9adebe9ff9bc4178dbcad2121a91d821b4308d107"
 
   bottle :unneeded
 
+  conflicts_with "cheat", :because => "Both install a `cheat` executable" if build.with?("all-tools") || build.with?("cheat")
```

Comment:
This needs to be below all the options because the `all-tools` and `cheat` options don't exist yet up here so the `if` clause will always be false.